src/views/virtualmachines/actions/VirtualMachineActionFactory.tsx

```diff
@@ -92,13 +94,16 @@ export const VirtualMachineActionFactory = {
         k8sPatch({
           data: [
             { op: 'replace', path: '/spec/rollback', value: true },
-            { op: 'remove', path: '/spec/migrateState' },
+            ...(currentStorageMigration?.spec?.migrateState
```

Comment:
Just to confirm the use case: if the `migrationState` was already removed and we try to remove it again then the server rejects the request with `422` ?